clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -108,8 +108,10 @@ export class CopybookDownloadService {
     documentUri: string,
     copybookNames: CopybookName[],
   ): Promise<boolean> {
-    if (await E4ECopybookService.getE4EClient(documentUri)) {
-      return true;
+    const e4eApi = await E4ECopybookService.getE4EAPI();
```

Comment:
How does this change resolve anything given that the `getE4EClient` does essentially the same stuff?

https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/dbce6eefd99477fe9882fb7ab77d329cebe1d60f/clients/cobol-lsp-vscode-extension/src/services/copybook/E4ECopybookService.ts#L54-L60

Reply:
file is e4e element but for some reason client can't be established. e.g invalid credentials / invalid configuration. It is there to not to check zowe related stuff.